bucket/odin.json

```
@@ -0,0 +1,30 @@
+{
+    "version": "dev-2022-03",
+    "description": "Odin is a general-purpose programming language with distinct typing, built for high performance, modern systems, and built-in data-oriented data types.",
+    "homepage": "https://odin-lang.org/",
+    "license": "BSD-3-Clause License",
```

```suggestion
    "license": "BSD-3-Clause",
```
See [SPDX identifiers](https://spdx.org/licenses/)